src/program.js

```diff
@@ -580,6 +580,13 @@ Example: $0 --help run.
         demandOption: false,
         type: 'string',
       },
+      'profile-create-new': {
+        describe: 'Create directory if the profile directory specified by ' +
+                  'the -p option doesn\'t already exist. If it already ' +
+                  'exists, an error will be thrown.',
```

Comment:
nit: `Create the profile directory if it does not already exist.`

This version does also omit the -p option, mainly because:
- that is only the option for the Firefox profile, and this is going to cover the chromium profile cli option as well
- I don't think we need to describe that here in too much detail (and option name may actually be clear enough for the user to guess)
- we should still add a more detailed description for the option in the [mozilla/extension-workshop](https://github.com/mozilla/extension-workshop) repository, as part of the web-ext cli options doc page (and we will be happy if you want to create a followup PR in that repo to take care of that after this is landed and we are about to release it in a new web-ext version)

Reply:
Sure. That sounds good! 👍